dev/snippets/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   dart_style: 2.0.2
   meta: 1.7.0
   platform: 3.0.0
+  process: 4.2.1
```

Comment:
Don't you also need to update all of the other pubspecs with `flutter update-packages --force-upgrade`? Or did you do that, and there just weren't any updates?

Reply:
i did